desktop/src/main/java/bisq/desktop/main/dao/bonding/bonds/BondsView.java

```diff
@@ -185,8 +185,11 @@ public void updateItem(final BondListItem item, boolean empty) {
                 };
             }
         });
-
         tableView.getColumns().add(column);
+        column.setComparator(Comparator.comparing(BondListItem::getAmount));
+        column.setSortType(TableColumn.SortType.ASCENDING);
+        tableView.getSortOrder().add(column);
```

Comment:
Don't change the default sort order to amount. Atm it is sorted by lockup date and it should stay like that. And all this doesn't work at all as desired as you have applied sorting to a string value which is the default behavior.